schunk_lwa4d_moveit_config/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(lwa4d_moveit_config)
```

Comment:
Did you do a file rename or did you use `git mv`?
As the project is still called `lwa4d_moveit_config`....this is not valid...
There are prably other references to the "old" package name...